### ISSUE TITLE
When trying to install I'm seeing: No compatible version found: web3-utils@^1.1.0

### ISSUE THREAD

**Issue by jackson-sandland** [warren-bank/ethereumjs-tx-sign#2]
Title says it all.

**Comment by warren-bank** [comment]
that's weird.. neither [package.json](https://github.com/warren-bank/ethereumjs-tx-sign/blob/master/package.json) nor, more importantly, [package-lock.json](https://github.com/warren-bank/ethereumjs-tx-sign/blob/master/package-lock.json) have any dependency on a module named "web3-utils".

I'll quickly run an `npm install --save '@warren-bank/ethereumjs-tx-sign'` and see if I can reproduce your error.. brb..

**Comment by warren-bank** [comment]
(sigh of relief).. no errors on my end:

```bash
c:\temp>npm install --save "@warren-bank/ethereumjs-tx-sign"
npm notice created a lockfile as package-lock.json. You should commit this file.

+ @warren-bank/ethereumjs-tx-sign@2.0.0
added 10 packages in 4.447s

   ?─────────────────────────────────────?
   │                                     │
   │   Update available 5.6.0 → 5.8.0    │
   │       Run npm i npm to update       │
   │                                     │
   ?─────────────────────────────────────?
```

**Comment by warren-bank** [comment]
Would it be fair to guess that your project contains the dependency on "web3-utils"?

If so, I'm thinking that maybe you should change the semversion in your _package.json_.. your error sounds like you're trying to install a version (ie: >= 1.1.0) that doesn't exist in the npm registry.

**Comment by warren-bank** [comment]
bingo..

according to the [npm registry](https://www.npmjs.com/package/web3-utils), the most recently published version of "web3-utils" is: `1.0.0-beta.33`
